### ISSUE TITLE
Ventura please

### ISSUE THREAD

**Issue by ImGGAAVVIINN** [foxlet/macOS-Simple-KVM#583]
Add Ventura support if you can. It will be greatly appreciated.
:D

**Comment by MobCode100** [comment]
You could try to download [osx-kvm](https://github.com/kholia/OSX-KVM) Ventura version that has just released today!

I expect ventura to be widely supported in the near future.

I successfully installed Ventura from OSX-KVM repo, having to add `+avx2` to the `MY_OPTIONS` line and changing `-cpu Penryn` to `-cpu host` in `OpenCore-Boot.sh` before installation, or else you might get errors relating to dyld cache / missing library files (e.g. libSystem.B.dylib). Make sure your CPU model supports avx2 instruction.

Edit: The above solution is no longer required for OSX-KVM as the problem has been fixed so just proceed with normal installation **however, depending on your computer you might need to do it if your other device requires avx2 on the CPU** such as GPU, according to [here](https://github.com/kholia/OSX-KVM/pull/207#issuecomment-1426707658).

@ImGGAAVVIINN You can install [OneClick macOS SImple KVM](https://oneclick-macos-simple-kvm.notaperson535.is-a.dev/docs/start-here) if you prefer a fork macOS-Simple-KVM. Honestly, you should stop creating issues here as this project is considered dead. It is better to create issues for other currently maintained and updated projects.

**Comment by bhat-sahab** [comment]
@MobCode100 Is +avx2 required to while installation or we can add it later? I add +avx2 to the MY_OPTIONS, but getting same issue, though I added it after installation had completed.

**Comment by MobCode100** [comment]
> @MobCode100 Is +avx2 required to while installation or we can add it later? I add +avx2 to the MY_OPTIONS, but getting same issue, though I added it after installation had completed.

You must do it before installing/upgrading because the installer reads the hardware and initialize the proper storage/boot device to be rebooted later. I'm not sure in detail how this exactly works, but that's what I did to make the installation successful. Make sure that your CPU supports avx2 instruction by running `lscpu | grep avx2`.

**Comment by bhat-sahab** [comment]
@MobCode100 
I tried it again with avx2 enable initially and same it gets stuck on same kernel panic with missing libSystem.B.dylib.
I have i5 12400f with windows 11 running ubuntu 20 under wsl2.

**Comment by MobCode100** [comment]
> @MobCode100 I tried it again with avx2 enable initially and same it gets stuck on same kernel panic with missing libSystem.B.dylib. I have i5 12400f with windows 11 running ubuntu 20 under wsl2.

Wierd. Remove OSX-KVM folder and redo the cloning process and everything. After launching with `OpenCore-Boot.sh`, at the screen where you have to erase the disk and reinstall macOS, click utilities at the top bar, run terminal, and enter `sysctl -a | grep avx`. If you have it enabled, it will show the output `hw.optional.avx2_0: 1`

**Comment by bhat-sahab** [comment]
> Wierd. Remove OSX-KVM folder and redo the cloning process and everything. After launching with OpenCore-Boot.sh, at the screen where you have to erase the disk and reinstall macOS, click utilities at the top bar, run terminal, and enter sysctl -a | grep avx. If you have it enabled, it will show the output hw.optional.avx2_0: 1

Tried the command in the recovery mode terminal and its enabled.

![Screenshot 2022-10-29 131029](https://user-images.githubusercontent.com/26200610/198820056-4884fedb-7eba-4f4b-9b2e-5a3e08257260.png)

**Comment by MobCode100** [comment]
> Tried the command in the recovery mode terminal and its enabled.

Run this `lscpu | grep avx2` on your Ubuntu terminal, does it show anything? If it's not try, to enable avx2 in Windows. I found a [video](https://www.youtube.com/watch?v=DQ8QI3S-mzs) for that, not sure if it works well.

**Comment by bhat-sahab** [comment]
> Run this `lscpu | grep avx2` on your Ubuntu terminal, does it show anything? If it's not try, to enable avx2 in Windows. I found a [video](https://www.youtube.com/watch?v=DQ8QI3S-mzs) for that, not sure if it works well.

Tried that already its enabled in wsl, im trying big sur just to check if everything is fine.

**Comment by MobCode100** [comment]
> Tried that already its enabled in wsl, im trying big sur just to check if everything is fine.

Try changing `-cpu Penryn` to `-cpu host`. That should fix the issue. I've updated my original comment.

**Comment by bhat-sahab** [comment]
> Try changing `-cpu Penryn` to `-cpu host`. That should fix the issue. I've updated my original comment.

Well this worked lol. Thanks!

**Comment by ash-hashtag** [comment]
> > Tried that already its enabled in wsl, im trying big sur just to check if everything is fine.
> 
> Try changing `-cpu Penryn` to `-cpu host`. That should fix the issue. I've updated my original comment.

If you don't mind, could you help?

Hey I've changed this, but I'm stuck at couldn't alloc class applekeystore
![image](https://user-images.githubusercontent.com/94179580/200239677-baabf795-1197-4c10-9555-359481d5e87e.png)

TLDR;
I want xcode(latest) for ios developing, I have windows 11, so I tried with VMs, didn't work, then with wsl and docker-osx, able to install catalina, but it was very unresponsive, so dual booted with linux mint, and I got monterey up and running, but I underestimated how much storage it gonna take, qemu was not opening because of unsufficient space, but the partitions order is not well, to expand the size easily, so I tried meh, let's just clear everything and reinstall linux, and after I did, with more than enough storage, nope it's not working, I don't know what went wrong, then after looking for solutions, most of them are working on ubuntu, so switched to ubuntu, hope it works, no luck, I;ve tried big sur, it works, and tried to upgrade from the vm, but it failed, I would've probably gave up by this now if it didn't worked the first time, I haven't changed anything in bios, it just not working now, been messing with a lot of workarounds like from a week, but couldn't get it to work :(, monterey is enough for the latest xcode as of now, but I need it as soon as possible

Thanks

**Comment by cestcela** [comment]
I could not install Ventura with OSX-KVM, on HP 15 AMD Ryzen 5 5500U with Radeon Graphics, and Linux Mint 21.1. It stops after the last install, just like on Virtualbox and vmWare, or Docker OSX. Any idea please?

**Comment by MobCode100** [comment]
> I could not install Ventura with OSX-KVM, on HP 15 AMD Ryzen 5 5500U with Radeon Graphics, and Linux Mint 21.1. It stops after the last install, just like on Virtualbox and vmWare, or Docker OSX. Any idea please?

Can you describe the error? Or can you show the error messages?

**Comment by cestcela** [comment]
Here is a snapshot.
![Screenshot_2023-01-20_17-52-41](https://user-images.githubusercontent.com/73157060/213757554-8da742ba-cfad-424e-8bb1-c8b5e77c5b49.png)

**Comment by cestcela** [comment]
Normally -with Big Sur or Monterey- once the install is finished, you would just boot on the disk you erased at the disk utility, and the OS would boot. But here it seems that Ventura gets to know it's an AMD and cannot boot.

**Comment by cestcela** [comment]
Normally -with Big Sur or Monterey- once the install is finished, you would just boot on the disk you erased at the disk utility, and the OS would boot. But here it seems that Ventura gets to know it's an AMD and 
![Screenshot_2023-01-20_17-52-41](https://user-images.githubusercontent.com/73157060/213784769-68749bb9-c9ad-499c-841e-63894ed57924.png)
cannot boot.

**Comment by MobCode100** [comment]
> Normally -with Big Sur or Monterey- once the install is finished, you would just boot on the disk you erased at the disk utility, and the OS would boot. But here it seems that Ventura gets to know it's an AMD and cannot boot.

That's strange, usually i have problem with that message because of the internet connection. While I'm looking for solution, could you try [this](https://github.com › notAperson535
OneClick-macOS-Simple-KVM - GitHub) and tell me if you can run ventura.

**Comment by cestcela** [comment]
I'm trying to install with HFS+ journaled at the moment. I know the same AMD problem happens with Windows desktop also. I tried to upgrade from Catalina to Ventura and it failed also. DrDonk could not find a solution to this issue: https://github.com/DrDonk/unlocker/issues/47

**Comment by cestcela** [comment]
HFS+ journaled failed. Do not worry about it, it seems there is no fix.

**Comment by MobCode100** [comment]
Hey, that's nice. At least you have a lead. It's easier to troubleshoot when you have debug message like that. Good luck with it. Although I find it weird as latest osx-kvm has fixed the avx2 issue by supporting non avx2 CPU, could be different case for your CPU. Try tinker with the MY_OPTIONS, changing Penrynn to a new generation might help.

**Comment by cestcela** [comment]
Following your advice I put +avx2, and I changed -cpu Penryn to -cpu Skylake-Client which I got from https://github.com/kholia/OSX-KVM/blob/master/notes.md
It went on well until the PKDownloadError error 8 arose.
![Screenshot_2023-01-21_21-05-12](https://user-images.githubusercontent.com/73157060/213885783-0573e49c-ed99-49c3-8acc-22eb0f5ce72e.png)
happened. I start again now (9.20 pm here) and we shall see during the night.

**Comment by cestcela** [comment]
sounds good at 0.00am
![last Screenshot_2023-01-21_23-59-54](https://user-images.githubusercontent.com/73157060/213890413-90a3f9ed-f0db-4522-ab1e-30bf446a5153.png)

**Comment by cestcela** [comment]
macOS Ventura in OSX-KVM -QEMU/KVM. with OpenCore- now working on AMD!!! 
 
Distro: Linux Mint 21.1 Vera. Desktop: Xfce. 
AMD 
![LAST Screenshot_2023-01-22_01-33-35](https://user-images.githubusercontent.com/73157060/213895348-b7c1c718-ed18-4190-8a90-3469b4755240.png)
Ryzen 5 5500U with Radeon Graphics

**Comment by MobCode100** [comment]
> macOS Ventura in OSX-KVM -QEMU/KVM. with OpenCore- now working on AMD!!! 
>  
> Distro: Linux Mint 21.1 Vera. Desktop: Xfce. 
> AMD 
> ![LAST Screenshot_2023-01-22_01-33-35](https://user-images.githubusercontent.com/73157060/213895348-b7c1c718-ed18-4190-8a90-3469b4755240.png)
> Ryzen 5 5500U with Radeon Graphics

Congrats @cestcela , mind telling me what did you do to solve this? It would be very helpful to others that have the similar problem too.

**Comment by cestcela** [comment]
From the above section, on Oct 27, 2022 you commented: "+avx2 to the MY_OPTIONS line and changing -cpu Penryn to -cpu host in OpenCore-Boot.sh before installation"

Which to suit the AMD case had to become: +avx2 to the MY_OPTIONS line and changing -cpu Penryn to -cpu Skylake-Client in OpenCore-Boot.sh before installation

Thanks to Kholia's work: https://github.com/kholia/OSX-KVM/blob/master/notes.md

Kholia+MobCode100=Ventura now working on AMD Ryzen 
(OSX-KVM -QEMU/KVM- with OpenCore)
